server/edusync/src/main/java/com/codestates/edusync/model/study/studygroup/controller/StudygroupController.java

```diff
@@ -111,7 +111,7 @@ public ResponseEntity getStudygroupPage(@RequestParam("page") @Positive Integer
                                             @RequestParam("size") @Positive Integer size){
 
         Page<Studygroup> studygroupPage = studygroupService.getWithPaging(page-1, size);
-        List<Studygroup> studygroupList =
+        List<Studygroup> studygroupList = // Stream 내부에서 Service 다량 쿼리 발생, 로직 수정 필요
```

Comment:
```suggestion
        List<Studygroup> studygroupList = // fixme: Stream 내부에서 Service 다량 쿼리 발생, 로직 수정 필요
```
fixme로 기록해두면 나중에 찾아보기 좋을 것 같습니다.